contracts/helpers/TransferHelper.sol

```diff
@@ -131,6 +182,11 @@ contract TransferHelper is TransferHelperInterface, TokenTransferrer {
                 )
             );
 
+            // Check that a conduit exists at the derived address.
+            if (conduit.codehash != _CONDUIT_RUNTIME_CODE_HASH) {
```

Comment:
This check is unnecessary; instead, we should retrieve the magic value returned when making the call to the conduit and compare it to the expected value, throwing if it does not equal that value